UI/window-basic-settings.cpp

```diff
@@ -3384,6 +3461,85 @@ void OBSBasicSettings::ReloadAudioSources()
 	LoadAudioSources();
 }
 
+void OBSBasicSettings::MultichannelToggled(bool state)
+{
+	if (state)
+	{ //display all channels
+		ui->channelSetup->addItem("2.1");
+		ui->channelSetup->addItem("4.0 Quad");
+		ui->channelSetup->addItem("4.1");
+		ui->channelSetup->addItem("5.1");
+		ui->channelSetup->addItem("7.1");
+		blog(LOG_INFO, "Multichannel ON, bitrates > 320 kbs unlocked");
+	//display all bitrates
+		PopulateAACBitrates({ ui->simpleOutputABitrate,
+			ui->advOutTrack1Bitrate, ui->advOutTrack2Bitrate,
+			ui->advOutTrack3Bitrate, ui->advOutTrack4Bitrate,
+			ui->advOutTrack5Bitrate, ui->advOutTrack6Bitrate });
+	}	
+	else
+	{ 
+		blog(LOG_INFO, "Multichannel OFF");
+	/* reset channel and bitrates settings when Multichannel is switched OFF*/
+		/* reset channel menu and save new setting */
+		int idx_channel = ui->channelSetup->currentIndex();
+		if (idx_channel >= 2)
+		{
+			ui->channelSetup->setCurrentIndex(1);
+			ui->channelSetup->setProperty("changed", QVariant(true));
+			config_set_string(main->Config(), "Audio", "ChannelSetup",
+				"Stereo");
+		}
+		for (int i = 6; i >= 2; i--) ui->channelSetup->removeItem(i);
+		blog(LOG_INFO, "Channel list reset to mono & stereo");
+
+		/* reset audio bitrate for simple and adv mode, update list of bitrates
+		and save setting */
+
+		blog(LOG_INFO,
+			"Bitrates restricted to under 320kbs");
+		blog(LOG_INFO,
+			"If current bitrate > 320 kbs, bitrate reset to 160 kbs");
+		restrictresetBitrates({ ui->simpleOutputABitrate,
+			ui->advOutTrack1Bitrate, ui->advOutTrack2Bitrate,
+			ui->advOutTrack3Bitrate, ui->advOutTrack4Bitrate, 
+			ui->advOutTrack5Bitrate, ui->advOutTrack6Bitrate}, 320);
+
+		SaveCombo(ui->simpleOutputABitrate, "SimpleOutput", "ABitrate");
+		SaveCombo(ui->advOutTrack1Bitrate, "AdvOut", "Track1Bitrate");
+		SaveCombo(ui->advOutTrack2Bitrate, "AdvOut", "Track2Bitrate");
+		SaveCombo(ui->advOutTrack3Bitrate, "AdvOut", "Track3Bitrate");
+		SaveCombo(ui->advOutTrack4Bitrate, "AdvOut", "Track4Bitrate");
+		SaveCombo(ui->advOutTrack5Bitrate, "AdvOut", "Track5Bitrate");
+		SaveCombo(ui->advOutTrack6Bitrate, "AdvOut", "Track6Bitrate");
+	}
+}
+
+/* resets current bitrate if too large and restricts the number of bitrates 
+displayed when multichannel OFF */
+
+void restrictresetBitrates(initializer_list<QComboBox*> boxes,
+	int maxbitrate)
+{
+	for (auto box : boxes) {
+		int idx = box->currentIndex();
+		int max_bitrate = FindClosestAvailableAACBitrate(maxbitrate);
+		int count = box->count();
+		int max_idx = box->
+			findText(QT_UTF8(std::to_string(max_bitrate).c_str()));
```

Comment:
Style: Indent with two tabs to continue a line.

That said, I'm not sure of the best place to place the line break for this.